CHANGELOG.md

```diff
@@ -2,6 +2,87 @@
 
 About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
 
+## Release v5.0.3 - 2024/03/NN
+
+### News
+
+* Update fluentd to 1.16.4
+* Update bundled gems
+* msi: fixed wrong environment path for Fluent Package Prompt (#606)
+  * It breaks fluent-diagtool behavior to launch fluent-gem correctly.
+* msi: removed unnecessary path delimiter (#607)
+  * It doesn't cause any problem yet, but it should treat `%~dp0~ correctly.
```

Comment:
Perhaps, did you all forget to add ``"`%~dp0~ with"`` ``"`"`` to emphasize here?

Reply:
Is this correct?

```suggestion
  * It doesn't cause any problem yet, but it should treat `%~dp0` correctly.
```

Reply:
Thanks! I've fixed it.